.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
I think we should add `.idea/.gitignore` to the top level `.gitignore`.

Reply:
For now I removed it, I will add it later on when it creates again